test/e2e/onLCP-test.js

```diff
@@ -282,6 +264,9 @@ describe('onLCP()', async function () {
     // Wait for a frame to be painted.
     await browser.executeAsync((done) => requestAnimationFrame(done));
 
+    // Wait a bit to ensure beacon gets a chance to be sent
+    await browser.pause(1000);
```

Comment:
Can you explain why this is needed? (At minimum it seems like the comment is incorrect, since there's already code to wait for the beacon below.)

Reply:
No I can't! Still digging but was gonna ask if you have any ideas, cause this has tests has definitely been flakey with no beacon being sent, and this pause seems to fix it.

Was wonder if the move to only reporting LCP `whenIdle` made this more flakey on CI. And if there's a better way. Can update comment to that if think it is that.

Reply:
Maybe instead I could change line above from this:

```js
await navigateTo('/test/lcp?imgDelay=0&imgHidden=1');
```
to this:
```js
await navigateTo('/test/lcp?imgDelay=0&imgHidden=1', {readyState: 'interactive'});
```
?

Reply:
Yeah, you could try the latter to see if that fixes things. I believe the code to wait for a frame to paint is meant to ensure the page is visible before hiding it, but it's possible FF fires rAF before that?

Reply:
It was actually Chrome that was flakey with this one. This PR has expended beyond it's initial scope and has since been renamed!

Reply:
That seems to have worked and no downside to that I don't think, and seems better. So let's go with that.